### PR TITLE
fix oracle table name must to upper

### PR DESCRIPTION
**Pull request by yu199195**

### CODE REVIEW

File: shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/model/physical/jdbc/handler/TableNamePatternHandler.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import org.apache.shardingsphere.infra.spi.typed.TypedSPI;
+
+/**
+ * Table name pattern handler.
+ */
+public interface TableNamePatternHandler extends TypedSPI {
+    
+    /**
+     * decorate table name pattern.
```
Comment (terrymanu) [inline shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/model/physical/jdbc/handler/TableNamePatternHandler.java:28]: Please upper case for first letter of java doc

File: shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/model/physical/jdbc/handler/TableNamePatternHandler.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+     * 
+     * @param tableNamePattern table name pattern
+     * @return decorate of table name pattern
```
Comment (terrymanu) [inline shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/model/physical/jdbc/handler/TableNamePatternHandler.java:31]: The java doc `decorate of table name pattern` is not good sentence,
It is better to revise it as `decorated table name pattern`

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/model/physical/jdbc/handler/TableNamePatternHandler.java (hunk 1 above)
Comment (yu199195) [reply shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/model/physical/jdbc/handler/TableNamePatternHandler.java]: modify already

File: shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/model/physical/jdbc/handler/TableNamePatternHandler.java (hunk 2 above)
Comment (yu199195) [reply shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/model/physical/jdbc/handler/TableNamePatternHandler.java]: modify already

**Review by terrymanu — APPROVED** [review]

**Pull request merged**